GpsDataAnalyzer/visualizer/visualizer.py

```diff
@@ -1,22 +1,27 @@
 """
 Usage: visualizer.py
 
-Parse the xml or csv file and generate a GpsDataSet
+Visualize the classified data as histogram and line graph with min/max/mean/std/availability information
 """
 import matplotlib.pyplot as plt
 import matplotlib.dates as dates
 import numpy as np
-import seaborn as sns
 import pandas as pd
 import os
 
 from datetime import datetime
 from datetime import timedelta
 from datetime import timezone
 
+"""
+Deviation distribution zone
+"""
+FIVE_METERS = 5
```

Comment:
Would `HIGH/LOW_CONFIDENCE_THRESHOLD` be more representative? Just thinking in case one wants to change the values down the line, either way is fine for now.

Reply:
Right, that makes more sense. Will do the change. Thanks!